aws/v13.1.0/release.yaml

```
@@ -0,0 +1,64 @@
+# Generated with:
+# devctl release create --name 13.1.0 --provider aws --base 13.0.0 --component aws-operator@9.3.8 --app cert-manager@2.4.1 --app external-dns@2.1.1 --component cluster-operator@3.5.1 --app cert-exporter@1.5.0 --app chart-operator@2.6.0 --app kiam@1.7.0 --app metrics-server@1.2.1 --component app-operator@3.2.0 --component aws-cni@1.7.8 --component containerlinux@2605.12.0 --component etcd@3.4.14 --component kubernetes@1.18.15 --app node-exporter@1.7.1 --overwrite
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v13.1.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.5.0
+  - name: cert-manager
+    version: 2.4.1
+  - name: chart-operator
+    version: 2.6.0
+  - componentVersion: 1.18.1
+    name: cluster-autoscaler
+    version: 1.18.3
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - name: external-dns
+    version: 2.1.1
+  - name: kiam
+    version: 1.7.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.0
+  - name: metrics-server
+    version: 1.2.1
+  - name: net-exporter
+    version: 1.9.2
+  - name: node-exporter
+    version: 1.7.1
+  components:
+  - name: app-operator
+    releaseOperatorDeploy: true
+    version: 3.2.0
+  - name: aws-cni
+    version: 1.7.8
+  - name: aws-operator
+    releaseOperatorDeploy: true
+    version: 9.3.8
+  - name: calico
+    version: 3.15.3
+  - name: cert-operator
+    reference: 0.1.0-2
+    releaseOperatorDeploy: true
+    version: 0.1.0
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 3.5.1
```

```suggestion
    version: 3.6.0
```
If it's okay, let's include the cluster status fix in the release.